doc/source/whatsnew/v0.23.0.rst

```diff
@@ -31,7 +31,7 @@ Check the :ref:`API Changes <whatsnew_0230.api_breaking>` and :ref:`deprecations
 .. warning::
 
    Starting January 1, 2019, pandas feature releases will support Python 3 only.
-   See `Dropping Python 2.7 <https://pandas.pydata.org/pandas-docs/version/0.24/install.html#install-dropping-27>`_ for more.
+   See :ref:`Dropping Python 2.7 <install>` for more.
```

Comment:
You will need to keep the absolute reference here as well, since this needs to refer to the versioned docs (that page doesn't exist anymore on the stable docs)